server/course/course.controller.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+const {
```

Comment:
Course controller is a big one, maybe it would be good idea to separate active user controller & router in separate module and just hook it up on course router

@ExtensionEngine/tailor What do you think?